README.md

```diff
@@ -3,49 +3,84 @@
 This repository contains TypeScript implementations for parts of the LionWeb standard/specification.
 Those are contained in a number of NPM packages in the directory [`packages`](./packages) (in order of importance):
 
-* `core`
+- `core`
   The "core stuff" such as: base types, the LionCore M3 (including the `builtins` language), and (de-)serialization.
-* `utilities`
+- `utilities`
   Utilities on top of the `core` packages that might be broadly useful, but should not go into the `core` package.
-* `test`
+- `test`
   A package containing (unit) tests for the packages above.
-* `cli`
+- `cli`
   A package with an executable to trigger some of the functionality in `utilities` through a commandline interface (CLI), i.e. from the commandline.
-* `artifacts`
+- `artifacts`
   A package that generates artifacts (serialization chunks, diagrams, JSON Schemas) from some of the models constructed in the `core` and `test` packages.
 
 The `core`, `utilities`, and `cli` packages are published in the scope of [the `lionweb[package.json](package.json)` organization](https://www.npmjs.com/org/lionweb), meaning that they're all prefixed with `@lionweb/`.
 The other packages are for internal use only.
 
 Each of these packages have their own `README.md`.
 
+## Environment dependencies
 
-## Development dependencies
-
-* [Node.js](https://nodejs.org/): JavaScript runtime, version (at least) v18.18.0 (the LTS version as of 2023-09-29)
-  * NPM (bundled with Node.js): version (at least) 9.6.7
-* (optional) [PlantUML](https://plantuml.com/).
+- [Node.js](https://nodejs.org/): JavaScript runtime, version (at least) v18.18.0 (the LTS version as of 2023-09-29)
+  - NPM (bundled with Node.js): version (at least) 9.6.7
+- (optional) [PlantUML](https://plantuml.com/).
   An IDE plugin such as the one [for IntelliJ IDEA](https://plugins.jetbrains.com/plugin/7017-plantuml-integration) also does the trick.
 
 Note that development tends to be done with the latest Node.js and NPM versions.
 Currently, these are v19.9.0 and 9.8.1.
 
+## Containerized development environment
+
+If you prefer not to install the development dependencies on your machine, you can use our containerized development environment for the LionCore TypeScript project. This environment provides a consistent and isolated development environment that is easy to set up and use. To get started, follow the instructions in our [containerized development environment guide](./documentation/dev-environment.md). However, you can streamline the process by running the following command:
+
+```sh
+docker run -it --rm --net host --name working-container -v ${PWD}:/work indamutsa/lionweb-devenv:v1.0.0 /bin/zsh
```

Comment:
Is `indamutsa/lionweb-devenv:v1.0.0` a specific name, or a name that can be chosen by the dev? It might be good to make that explicit.

Reply:
It's the image I created that includes all the essential components required to set up the environment for running the project. If you prefer, we have the option to migrate this to LionWeb's Docker repository for centralized management and easier access.

Reply:
Do we have a LionWeb Docker repository already? I don't anyone set that up so far, so this is fine. I just wanted to check that it's not a "random" name, but matches something existing elsewhere.